src/twisted/application/runner/test/test_runner.py

```diff
@@ -398,19 +425,19 @@ class DummyKill:
     what arguments it was given.
     """
 
-    def __init__(self):
-        self.calls = []
+    def __init__(self) -> None:
+        self.calls = []  # type: List[Tuple[int, int]]
 
-    def __call__(self, pid, sig):
+    def __call__(self, pid: int, sig: int) -> None:
         self.calls.append((pid, sig))
 
 
 class DummyStandardIO:
     """
-    Stub for L{sys} which provides L{BytesIO} streams as stdout and stderr.
+    Stub for L{sys} which provides L{StringIO} streams as stdout and stderr.
     """
 
-    def __init__(self, stdout, stderr):
+    def __init__(self, stdout: TextIO, stderr: TextIO) -> None:
```

Comment:
Should this be updated to `StringIO` as well?

Reply:
`StringIO` is the concrete class; `TextIO` is the type.  (This is unlike `BytesIO` where the names match…)

Reply:
You're right. I always use `IO[str]` myself, so I forgot about `TextIO`.